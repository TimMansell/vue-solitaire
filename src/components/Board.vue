<template>
  <div
    class="board"
    data-test="board">
    <Aces />
    <Columns />
  </div>
</template>

<script>
import Aces from '@/components/Aces.vue';
import Columns from '@/components/Columns.vue';

export default {
  name: 'Board',
  components: {
    Aces,
    Columns,
  },
};
</script>


<!-- Add "scoped" attribute to limit CSS to this component only -->
<style scoped lang="scss">
.board {
  padding: .5rem;
  margin: 0 auto;
  max-width: 960px;

  @media (min-width: 900px) {
    padding: 1rem;
  }
}
</style>
