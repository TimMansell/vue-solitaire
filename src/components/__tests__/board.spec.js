import { shallowMount } from '@vue/test-utils';
import Board from '@/components/Board.vue';

describe('Board.vue', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(Board);

    expect(wrapper).toMatchSnapshot();
  });
});
