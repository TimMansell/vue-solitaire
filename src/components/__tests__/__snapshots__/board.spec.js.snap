// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Board.vue matches snapshot 1`] = `
<div>
  <stats-stub></stats-stub>
  <div data-test="board" class="board">
    <foundation-stub></foundation-stub>
    <columns-stub></columns-stub>
  </div>
</div>
`;
